example/integration_test/setup_ui_test.dart

```diff
@@ -1,13 +1,18 @@
+import 'dart:io';
```

Comment:
I'm ok with the change, but in this case setup_ui_test file should be renamed to reflect it's the sdk setup functionalities which are tested (currently the title suggests it should check setupUI method)